<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSettings</class>
 <widget class="QDialog" name="DlgSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TBI Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="BoxURLs">
     <property name="title">
      <string>Base URLs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="LabelRK">
        <property name="text">
         <string>Rebuilding Kit:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="EditRK">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="ButtonResetRK">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="LabelRM">
        <property name="text">
         <string>Rebuilding Manual:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="EditRM"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="ButtonResetRM">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="LabelTip">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>&quot;%1&quot; will be replaced by the document number</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="BoxSearch">
     <property name="title">
      <string>Search</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="HLayoutSearch">
        <item>
         <widget class="QCheckBox" name="CheckRTSearch">
          <property name="text">
           <string>Real-time search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckWholeWordsOnly">
          <property name="text">
           <string>Whole words only</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="HLineSearch">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <widget class="QCheckBox" name="CheckSearchTechPub">
          <property name="text">
           <string>Tech. publication</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="CheckSearchReplaces">
          <property name="text">
           <string>Replaces</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="CheckSearchReleaseDate">
          <property name="text">
           <string>Release date</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="CheckSearchReplacedBy">
          <property name="text">
           <string>Replaced by</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="CheckSearchNumber">
          <property name="text">
           <string>Number</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="CheckSearchRegisteredBy">
          <property name="text">
           <string>Registered by</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="CheckSearchTitle">
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="CheckSearchCategory">
          <property name="text">
           <string>Category</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="LabelAdditionalSearch">
          <property name="text">
           <string>Additional search fields</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="CheckSearchRK">
          <property name="text">
           <string>RK number</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="CheckSearchNotes">
          <property name="text">
           <string>Notes</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="BoxCategories">
     <property name="title">
      <string>Categories list</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="HLayoutCategories">
        <item>
         <spacer name="HSpacerCategories1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonClearCategories">
          <property name="text">
           <string>Reset category items</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="HSpacerCategories2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="VSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutButtons">
     <item>
      <spacer name="HSpacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>EditRK</tabstop>
  <tabstop>ButtonResetRK</tabstop>
  <tabstop>EditRM</tabstop>
  <tabstop>ButtonResetRM</tabstop>
  <tabstop>CheckRTSearch</tabstop>
  <tabstop>CheckWholeWordsOnly</tabstop>
  <tabstop>CheckSearchNumber</tabstop>
  <tabstop>CheckSearchTitle</tabstop>
  <tabstop>CheckSearchCategory</tabstop>
  <tabstop>CheckSearchRK</tabstop>
  <tabstop>CheckSearchTechPub</tabstop>
  <tabstop>CheckSearchReleaseDate</tabstop>
  <tabstop>CheckSearchRegisteredBy</tabstop>
  <tabstop>CheckSearchReplaces</tabstop>
  <tabstop>CheckSearchReplacedBy</tabstop>
  <tabstop>ButtonClearCategories</tabstop>
  <tabstop>ButtonOK</tabstop>
  <tabstop>ButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
